ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java

```diff
@@ -414,6 +414,8 @@ private static void setState(
               context.getString(
                   boolValue ? R.string.state_on_description : R.string.state_off_description));
         }
+      } else if (state.equals("expanded") && value.getType() == ReadableType.Boolean) {
+        info.addAction(value.asBoolean() ? AccessibilityNodeInfoCompat.ACTION_EXPAND : AccessibilityNodeInfoCompat.ACTION_COLLAPSE); 
```

Comment:
Maybe I'm misreading this, but isn't this adding an action of "ACTION_EXPAND" when the state is "expanded"? 

It should do the opposite, when a state is "expanded" the only available action should be "ACTION_COLLAPSE" and when a state is "collapsed" the only available action should be "ACTION_EXPAND".

Reply:
Thanks a lot. I added the suggested changes. I also added these changes:
- updated the AccessibilityExample ([videos](https://github.com/facebook/react-native/pull/34353#issuecomment-1237601847))
- added `setTag` and `getTag` to retrieve and update the `accessibility_state_expanded` in ReactAccessibilityDelegate

https://github.com/facebook/react-native/blob/b80bcb3104c903f97974d90db3827b997af9c7cc/ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java#L268-L276

https://github.com/facebook/react-native/blob/b80bcb3104c903f97974d90db3827b997af9c7cc/ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java#L228-L235

- added the logic in performAccessibilityAction to announce the change in expanded collapsed state when using accessibility menu without implementation of [onAccessibilityAction](https://reactnative.dev/docs/next/view#onaccessibilityaction).

https://github.com/facebook/react-native/blob/b80bcb3104c903f97974d90db3827b997af9c7cc/ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java#L365-L371

- updated the summary with the new video test cases
- further investigated the issue with onAccessibilityAction not triggered on Fabric (summary available [here](https://github.com/reactwg/react-native-new-architecture/discussions/74#discussioncomment-3566068)).